### PR TITLE
Update model.js

### PR DESCRIPTION
**Pull request by crandmck**
Add initial doc comments for nestRemoting per #1767 
@Amir-61 Please fill in missing info, e.g. options.  Also, how is callback used?

### CODE REVIEW

File: lib/model.js [hunk 1]
```diff
@@ -660,6 +660,19 @@ module.exports = function(registry) {
...
+   * @param {String} relationName Name of the nested relation.
+   * @options {Object} [options] See below.
+   * @param {String} pathName 
+   * @param {String} filterMethod
+   * @param {String} paramName
+   * @param {String} getterName
+   * @param {Boolean} hooks Whether to inherit before/after hooks 
+   * @callback {Function} [cb] The callback function.
+   */
+
```
Comment (superkhau) [inline lib/model.js]: Remove this one space.

**Comment by Amir-61** [comment]
Hey @crandmck,

I added the missing info and re-based/squashed the commits to one single commit; however, a few of CI tests are failing, although they are all passing on local machine. I don't think if it has anything to do with this pull request, but just in case I leave it for you to merge it...

Thanks!
Amir

**Comment by crandmck** [comment]
@rmg This is a very small PR for API doc improvements but we have CI failures.

Can we merge this without any issue?

**Comment by rmg** [comment]
@crandmck agreed, CI shouldn't block this PR. Merge when you're ready.

**Pull request merged**